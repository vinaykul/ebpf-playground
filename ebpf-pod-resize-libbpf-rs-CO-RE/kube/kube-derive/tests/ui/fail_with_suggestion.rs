use kube_derive::CustomResource;
use schemars::JsonSchema;
use serde::{Deserialize, Serialize};

#[derive(CustomResource, Serialize, Deserialize, Debug, Clone, JsonSchema)]
#[kube(group = "clux.dev", version = "v1", kind = "Foo", shortnames = "foo")]
struct FooSpec {
    foo: String,
}

fn main() {}
