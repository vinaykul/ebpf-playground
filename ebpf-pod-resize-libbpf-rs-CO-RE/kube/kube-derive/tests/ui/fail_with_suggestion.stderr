error: Unknown field: `shortnames`. Did you mean `shortname`?
 --> $DIR/fail_with_suggestion.rs:6:58
  |
6 | #[kube(group = "clux.dev", version = "v1", kind = "Foo", shortnames = "foo")]
  |                                                          ^^^^^^^^^^
